packages/styling/src/styles/IconCodes.ts

```diff
@@ -2518,7 +2518,7 @@ export const IconCodes = {
   /**
    * Icon code with the value '\uE8AB'.
    */
-  switcher: '\uE8AB',
+  switch: '\uE8AB',
```

Comment:
Is this technically a patch since the name/contract is changing? Should the old names still be supported here until a major release? @dzearing, thoughts?